runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -869,6 +866,9 @@ Java_java_lang_invoke_MethodHandleNatives_resolve(JNIEnv *env, jclass clazz, job
 	}
 
 done:
+	j9mem_free_memory(name);
+	j9mem_free_memory(signature);
```

Comment:
Line 705 also does a `j9mem_free_memory(signature)` before a `goto done;` which will double free.

Please check for any other prior free calls covering these variables